sched/misc/assert.c

```diff
@@ -255,13 +271,29 @@ static void dump_task(FAR struct tcb_s *tcb, FAR void *arg)
 
   group_argvstr(tcb, args, sizeof(args));
 
+  /* get the task_state */
+
+  nxsched_get_stateinfo(tcb, state, sizeof(state));
+  if ((s = strchr(state, ',')) != NULL)
+    {
+      *s = '\0';
+      snprintf(str, sizeof(str), "%s  %s", state, s + 1);
```

Comment:
why use the two spaces

Reply:
because i need change  "waiting,mutex:9" -> "waiting  mutex:9" to Keep it consistent with ps

Reply:
one space is enough?

Reply:
Yes.